e2e/tests/interchain_accounts/incentivized_test.go

```diff
@@ -66,7 +65,7 @@ func (s *IncentivizedInterchainAccountsTestSuite) TestMsgSendTx_SuccessfulBankSe
 	chainBAccount := s.CreateUserOnChainB(ctx, testvalues.StartingTokenAmount)
 
 	t.Run("broadcast MsgRegisterInterchainAccount", func(t *testing.T) {
-		version := getICAVersion(testconfig.GetChainATag(), testconfig.GetChainBTag())
+		version := "" // allow version to be specified by the controller chain since both chains should support incentivized channels
```

Comment:
I set the version to empty string because the tests in this file are all supposed to be fee enabled and all supported versions of ibc-go (from v4 onwards) support ics29.